features/extension-mgt/org.wso2.carbon.identity.extension.mgt.feature/resources/extensions/connections/signicat/metadata.json

```diff
@@ -0,0 +1,272 @@
+{
+    "create": {
+        "image": "assets/images/logos/signicat.svg",
+        "header": "Signicat",
+        "Subheading": "Login users with eIDs.",
+        "documentation": "",
+        "modal": {
+            "form": {
+                "fields": [
+                    {
+                        "index": 0,
+                        "ariaLabel": "Signicat Name",
+                        "name": "name",
+                        "label": "Name",
+                        "type": "text",
+                        "required": true,
+                        "placeholder": "Enter a name for the connection.",
+                        "initialValue": "Signicat",
+                        "data-testid": "signicat-idp-create-wizard-content-idp-name",
+                        "maxLength": "50",
+                        "minLength": "3",
+                        "width": "13"
+                    },
+                    {
+                        "index": 1,
+                        "ariaLabel": "Client ID", 
+                        "type": "text", 
+                        "name": "clientId", 
+                        "label": "Client ID",
+                        "placeholder": "Enter Client ID.",
+                        "required": true, 
+                        "autoComplete": true,
+                        "data-testid": "signicat-idp-create-wizard-content-idp-client-id",
+                        "maxLength": "100",
+                        "minLength": "3",
+                        "width": "13"
+                    },
+                    {
+                        "index": 2,
+                        "ariaLabel": "Client Secret",
+                        "className": "addon-field-wrapper",
+                        "type": "password", 
+                        "name": "clientSecret", 
+                        "label": "Client secret",
+                        "placeholder": "Enter Client secret.",
+                        "required": true, 
+                        "autoComplete": true,
```

Comment:
```suggestion
```
Better to turn off the auto complete for secretes

Reply:
fixed